### ISSUE TITLE
Installing a plugin in APIGEE Templater

### ISSUE THREAD

**Issue by Gayathri071123** [apigee/apigee-templater#13]
Hope you are doing good.

I'm planning to add some changes in Flow Callout and Proxy plugins.
Could you please help me with the steps to install the plugin in APIGEE Templater.

Thanks for your support.

**Comment by tyayers** [comment]
Yes will update README with instructions to add or replace your own plugins.. will link here when complete.

**Comment by tyayers** [comment]
See example customized project here (customized cli with own plugins): https://github.com/tyayers/apigee-templater-custom
